bundle/bin/rke2-killall.sh

```diff
@@ -44,6 +44,7 @@ do_unmount() {
     if [ -n "${MOUNTS}" ]; then
         set -x
         umount ${MOUNTS}
+        rm -rf ${MOUNTS}
```

Comment:
I feel like this either shouldn't be recursive (there should not be anything left under these paths after the filesystems have been unmounted, or we should we use `--one-file-system` just to be extra safe, in case the unmount fails for some reason. I'd hate to accidentally delete a bunch of files out of a volume during a killall.